templates/taxi/car_confirm_delete.html

```diff
@@ -7,4 +7,4 @@ <h1>Delete car?</h1>
 
     <input type="submit" value="Yes" class="btn btn-danger">
   </form>
```

Comment:
As per the checklist, you should also add a `Cancel` button that leads to the previous page the user was on. This gives users a chance to change their mind before the deletion is finalized.